### DISCUSSION TITLE
Kubernetes version (> 1.25) support

### DISCUSSION THREAD

**Discussion by huangyingting** [chaos-mesh/chaos-mesh#4316] [General]
Chaos mesh states the supported kubernetes version is up to 1.25 per article https://chaos-mesh.org/supported-releases/
However, "Kubernetes 1.25 is being deprecated on January 14, 2024", wondering if there will be any updates on supported kuberenetes in future release?

**Comment by g1eny0ung** [comment]
Yep. We should update the support for recent Kubernetes versions. I saw the PR you opened. Let's take this further in https://github.com/chaos-mesh/chaos-mesh/pull/4319.
